firmware/examples/Gpio/source/main.cpp

```diff
@@ -0,0 +1,33 @@
+#include "L1_Drivers/gpio.hpp"
+
+int main(void)
+{
+    // Port 2, pin 0 must be connected to a switch that is able to ground and
+    // supply 3.3 volts to the pin (You can use a 1kOhm resistor to connect the
+    // pin to the switch for protection).
+    // Port 2, pin 2 must be connected to an LED that connects to ground (use
+    // the appropriate resistor to protect the LED).
```

Comment:
You may also want to link to an image of what this looks like.

Reply:
If I choose to use the on-board SJTwo led and button, would I still need to link an image?